data/.mlaunch_startup

```diff
@@ -0,0 +1,54 @@
+{
+"protocol_version": 2,
+"mtools_version": "1.6.4",
+"parsed_args": {
+"no_progressbar": false,
+"command": "init",
+"single": true,
+"replicaset": false,
+"nodes": 3,
+"arbiter": false,
+"name": "replset",
+"priority": false,
+"sharded": null,
+"config": 1,
+"csrs": true,
+"mongos": 1,
+"verbose": false,
+"port": 27017,
+"binarypath": null,
+"dir": "/Users/alenakhineika/www/mongosh/data",
+"hostname": "localhost",
+"auth": false,
+"username": "user",
+"password": "password",
+"auth_db": "admin",
+"auth_roles": [
+"dbAdminAnyDatabase",
+"readWriteAnyDatabase",
+"userAdminAnyDatabase",
+"clusterAdmin"
+],
+"auth_role_docs": false,
+"initial-user": true,
+"sslCAFile": null,
+"sslCRLFile": null,
+"sslAllowInvalidHostnames": false,
+"sslAllowInvalidCertificates": false,
+"sslMode": null,
+"sslPEMKeyFile": null,
+"sslPEMKeyPassword": null,
+"sslClusterFile": null,
+"sslClusterPassword": null,
+"sslDisabledProtocols": null,
+"sslAllowConnectionsWithoutCertificates": false,
+"sslFIPSMode": false,
+"sslClientCertificate": null,
+"sslClientPEMKeyFile": null,
+"sslClientPEMKeyPassword": null
+},
+"unknown_args": [],
+"startup_info": {
+"27017": "\"mongod\"  --dbpath \"/Users/alenakhineika/www/mongosh/data/db\" --logpath \"/Users/alenakhineika/www/mongosh/data/mongod.log\" --port 27017 --fork   --wiredTigerCacheSizeGB 1 "
+}
+}
```

Comment:
I think this file might have been added by accident?

Reply:
Removed.

Reply:
Looks like it’s still there, just so you know :)

Reply:
It should be gone now :)